networkx/algorithms/efficiency.py

```diff
@@ -147,4 +147,4 @@ def local_efficiency(G):
 
     """
     # TODO This summation can be trivially parallelized.
-    return sum(global_efficiency(nx.ego_graph(G, v)) for v in G) / len(G)
+    return sum(global_efficiency(nx.ego_graph(G, v, center = False)) for v in G) / len(G)
```

Comment:
```ego_graph``` is a lot of overhead for this simple case. Could it be replaced with:

    return sum(global_efficiency(G.subgraph(G[v])) / len(G)

Reply:
Yep, makes sense.